microservice/package/src/main/java/com/cumulocity/agent/packaging/BaseMicroserviceMojo.java

```diff
@@ -121,6 +121,16 @@ public abstract class BaseMicroserviceMojo extends AbstractMojo {
     @Parameter(property = "package.docker.baseImage", defaultValue = "alpine:3")
     protected String baseImage;
 
+    @Parameter(property = "pull.image.registry")
+    protected String pullRegistry;
+
+    @Parameter(property = "pull.image.username")
+    protected String pullUsername;
+
+    @Parameter(property = "pull.image.password")
+    protected String pullPassword;
```

Comment:
use the same prefix as `package.docker.baseImage`
```suggestion
    @Parameter(property = "package.docker.registry")
    protected String pullRegistry;

    @Parameter(property = "package.docker.username")
    protected String pullUsername;

    @Parameter(property = "package.docker.password")
    protected String pullPassword;
```